pkg/util/exec/exec.go

```diff
@@ -86,20 +86,20 @@ func startCommand(command string, arg ...string) (*exec.Cmd, io.ReadCloser, io.R
 	return cmd, stdout, stderr, err
 }
 
-func logOutput(stdout, stderr io.ReadCloser) {
+func logOutput(name string, stdout, stderr io.ReadCloser) {
 	if stdout == nil || stderr == nil {
 		logger.Warningf("failed to collect stdout and stderr")
 		return
 	}
 
-	// The logger for this exec package is responsible for logging output from other processes.
-	// Those child processes should appropriately be outputting at the desired global level.  Therefore,
+	// The child processes should appropriately be outputting at the desired global level.  Therefore,
 	// we always log at INFO level here, so that log statements from child procs at higher levels
 	// (e.g., WARNING) will still be displayed.  We are relying on the child procs to output appropriately.
-	if !logger.LevelAt(capnslog.INFO) {
+	childLogger := capnslog.NewPackageLogger("github.com/rook/rook", name)
```

Comment:
are we creating a new package logger object for each line of output? if so, it would be better to cache a package logger for each unique "name" that comes into this method.